servers/visual/shader_language.cpp

```diff
@@ -1767,10 +1767,10 @@ bool ShaderLanguage::_validate_function_call(BlockNode *p_block, OperatorNode *p
 
 	StringName name = static_cast<VariableNode *>(p_func->arguments[0])->name.operator String();
 
-	bool all_const = true;
+	// bool all_const = true;
```

Comment:
At this stage, it's probably better to remove it and kill the related `#if 0`'ed code.